mk/config.mk

```diff
@@ -582,3 +582,9 @@ CFG_SCMI_MSG_DRIVERS ?= n
 CFG_SCMI_MSG_CLOCK ?= n
 CFG_SCMI_MSG_RESET_DOMAIN ?= n
 CFG_SCMI_MSG_SMT ?= n
+
+# Path to the Python interpreter used by the build system.
+# By default the interpreter is defined by the system through /usr/bin/env.
```

Comment:
Is it? By default, `CFG_PYTHON` is set to `python3` so all scripts are invoked as `python3 foo.py`. In this case, I don't think that the hashbang line (`/usr/bin/env ...`) has any effect, does it?

Reply:
I agree the comment was a bit confusing or misleading. I was trying to refer to the behaviour prior to this patch which doesn't make sense. I updated the comment.